bin/client_package/installation/default

```diff
@@ -5,3 +5,6 @@ MYSTERIUM_CLIENT_TEQUILAPI_ADDRESS="localhost"
 
 # Port for listening incoming api requests
 MYSTERIUM_CLIENT_TEQUILAPI_PORT="4050"
+
+# Mysterium discovery address (default: http://testnet.mysterium.network:5000/v1)
```

Comment:
How `http://testnet.mysterium.network:5000/v1` is default? Isn't value in `.env` the default?